### PR TITLE
BCMOHAD-13649 || Created the New field, Updated the Page layout

### PR DESCRIPTION
**Pull request by RanadheerRG**
Created the New Field
Updated the Page layout
Updated the Permission Sets

### CODE REVIEW

File: MAiD - Case Management App/force-app/main/default/objects/Case/fields/If_Other_Race_Specify__c.field-meta.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>If_Other_Race_Specify__c</fullName>
+    <encryptionScheme>None</encryptionScheme>
```
Comment (cgiSunilM) [inline MAiD - Case Management App/force-app/main/default/objects/Case/fields/If_Other_Race_Specify__c.field-meta.xml:4]: @RanadheerRG: Please provide description and help text if provided. Or else, please check with Lara. Thanks!

File: MAiD - Case Management App/force-app/main/default/layouts/Case-MAiD Case Layout.layout-meta.xml [hunk 2]
```diff
@@ -427,11 +436,6 @@
             <actionType>QuickAction</actionType>
             <sortOrder>5</sortOrder>
         </platformActionListItems>
-        <platformActionListItems>
```
Comment (cgiSunilM) [inline MAiD - Case Management App/force-app/main/default/layouts/Case-MAiD Case Layout.layout-meta.xml:430]: @RanadheerRG, please add this quick action back, unless Lara asked us to remove it from the page layout.

**Review by cgiSunilM — CHANGES_REQUESTED** [review]

File: MAiD - Case Management App/force-app/main/default/layouts/Case-MAiD Case Layout.layout-meta.xml (hunk 2 above)
Comment (RanadheerRG) [reply MAiD - Case Management App/force-app/main/default/layouts/Case-MAiD Case Layout.layout-meta.xml:430]: It's already added in line 404

File: MAiD - Case Management App/force-app/main/default/objects/Case/fields/If_Other_Race_Specify__c.field-meta.xml (hunk 1 above)
Comment (RanadheerRG) [reply MAiD - Case Management App/force-app/main/default/objects/Case/fields/If_Other_Race_Specify__c.field-meta.xml:4]: Updated the help text

**Review by cgiSunilM — APPROVED** [review]
LGTM!

**Pull request merged**